services/core-api/app/api/mines/compliance/resources/compliance.py

```diff
@@ -23,7 +24,7 @@ def get(self, mine_no):
             raise NotFound("No mine record in CORE.")
 
         result = cache.get(NRIS_COMPLIANCE_DATA(mine.mine_no))
-        if result is None:
+        if result is None and Config.ENVIRONMENT_NAME == 'prod':
```

Comment:
Very nice!

Reply:
It makes tests fail unfortunately. 😭 
Not sure if it's worth keeping (although I wonder what those tests are actually testing because there's no way that they're getting real data in there)